src/managed/CommonManaged.props

```diff
@@ -6,8 +6,14 @@
   <PropertyGroup>
     <RepoRoot>$([System.IO.Path]::GetFullPath('$(MSBuildThisFileDirectory)../..'))/</RepoRoot>
     <VersionPrefix>2.0.0</VersionPrefix>
+    <!-- This is consistent with previously packaged assemblies, but it's not clear if it's what we want
+         going forward -->
+    <AssemblyFileVersion>2.0.0</AssemblyFileVersion>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
     <DisableImplicitFrameworkReferences>true</DisableImplicitFrameworkReferences>
+    <IncludeSymbols>true</IncludeSymbols>
+    <Serviceable>true</Serviceable>
+    <SkipVersionGeneration>true</SkipVersionGeneration>
   </PropertyGroup>
```

Comment:
Given that it appears like you want to skip our generation of assembly version info don't you also want to set GenerateAssemblyInfo?